app/components/video.js

```diff
@@ -53,7 +53,7 @@ export default function Video({ video }) {
           >
             <source src={video.url} type="video/mp4" />
             <p className="vjs-no-js">
-              <a href={video.url} target='_blank'>{ video.description }</a>
+              <a href={video.url} target='_blank' rel="external noopener noreferrer">{ video.description }</a>
```

Comment:
This is not external; and we don't want noreferrer here (and I think noopener is not required).

Reply:
I can remove the [external](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#attr-external) if you want but I'd leave it in.

The `noopener noreferrer` though is supposed to solve a security issue with target blank (https://owasp.org/www-community/attacks/Reverse_Tabnabbing) since it points to a different domain.